app/addons/verifyinstall/tests/componentsSpec.js

```diff
@@ -0,0 +1,136 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'react',
+  'testUtils',
+  'addons/verifyinstall/constants',
+  'addons/verifyinstall/components.react'
+
+], function (app, FauxtonAPI, React, testUtils, Constants, Components) {
+  FauxtonAPI.router = new FauxtonAPI.Router([]);
+
+  var assert = testUtils.assert;
+  var ReactTestUtils = React.addons.TestUtils;
+
+
+  describe('VerifyInstallResults', function () {
+    var container, el;
+
+    var tests = [
+      { key: 'CREATE_DATABASE', id: 'js-test-create-db' },
+      { key: 'CREATE_DOCUMENT', id: 'js-test-create-doc' },
+      { key: 'UPDATE_DOCUMENT', id: 'js-test-update-doc' },
+      { key: 'DELETE_DOCUMENT', id: 'js-test-delete-doc' },
+      { key: 'CREATE_VIEW', id: 'js-test-create-view' },
+      { key: 'REPLICATION', id: 'js-test-replication' }
+    ];
+
+    var testResults = {};
+    _.each(tests, function (test) {
+      testResults[Constants.TESTS[test.key]] = { complete: false };
+    });
+
+    afterEach(function () {
+      React.unmountComponentAtNode(container);
+    });
+
+    it('confirm all result fields blank before tests ran', function () {
+      container = document.createElement('div');
+      el = ReactTestUtils.renderIntoDocument(React.createElement(Components.VerifyInstallResults, {testResults: testResults}), container);
+
```

Comment:
Just a general remark: ReactTestUtils.renderIntoDocument ignores the container param - see https://github.com/facebook/react/blob/master/src/test/ReactTestUtils.js?source=c#L46